src/utils/sceneimagebuilder/sceneimagebuilder.cpp

```diff
@@ -0,0 +1,325 @@
+//========= -------------------------------------------------------------- ============//
+//
+// Purpose: 
+//
+//=====================================================================================//
+#include "appframework/AppFramework.h"
+#include "tier1/strtools.h"
+#include "tier0/icommandline.h"
+#include "tools_minidump.h"
+#include "filesystem_init.h"
+#include "filesystem_tools.h"
+#include "filesystem.h"
+#include "loadcmdline.h"
+#include "scriplib.h"
+#include "utlbuffer.h"
+#include "cmdlib.h"
+#include "sceneimage.h"
+#include "choreoevent.h"
+
+#include "sceneimagebuilder.h"
+
+
+//-----------------------------------------------------------------------------
+// Purpose: Global vars 
+//-----------------------------------------------------------------------------
+bool g_bBuilVcd  = false;
```

Comment:
Using global variables is a bad practise and should be avoided.
Reasons are:
1. Static initialization / shutdown order fiasko.
2. Hard to understand the program flow as globals can be read/written everywhere.

Just pack these into your app class.